skfda/misc/hat_matrix.py

```diff
@@ -25,10 +25,11 @@ class HatMatrix(
     RegressorMixin,
 ):
     """
-    Kernel estimators.
+    Hat Matrix.
 
-    This module includes three types of kernel estimators that are used in
-    KernelSmoother and KernelRegression classes.
+    Base class for :class:`~skfda.misc.hat_matrix.NadarayaWatsonHatMatrix`,
```

Comment:
Don't specify the subclasses in the description. You can add a "See also" field if you want to.